src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1967,6 +1967,29 @@
     /**
      * A left curly brace (<code>{</code>).
      *
+     * <p>For example:</p>
+     * <pre>
+     * class App{
+     * int num
```

Comment:
please add space before "{" and indent "int num" by 2 spaces.
This code is still for humans to read, formatting matters.